README.md

```diff
@@ -25,20 +25,56 @@ Another option is to download the repository and point to it using the `--track-
+
+| Parameter | Explanation | Type | Default Value |
+| --------- | ----------- | ---- | ------------- |
+| `number_of_replicas` | Number of index replicas | `int` | `0` |
+| `shard_count` | Number of primary shards | `int` | `2` |
+| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `8` |
+
 ### 2) bulk-size-evaluation
 
 This challenge performs bulk-indexing against a single index with varying bulk request sizes, ranging from 125 events/request to 50000 events/request.
 
+The table below shows the track parameters that can be adjusted along with default values:
+
+| Parameter | Explanation | Type | Default Value |
+| --------- | ----------- | ---- | ------------- |
+| `number_of_replicas` | Number of index replicas | `int` | `0` |
+| `shard_count` | Number of primary shards | `int` | `2` |
+| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `16` |
+
 ### 3) shard-sizing
 
 This challenge indexes 2 million events into an index consisting of a single shard 25 times. After each group of 2 million events has been inserted, 4 different Kibana dashboard configurations are benchmarked against the index. At this time no indexing takes place. There are two different dashboards being simulated, aggregating across 50% and 90% of the data in the shard.
 
 This challenge shows how shard sizing can be performed and how the nature of queries used can impact the results.
 
+The table below shows the track parameters that can be adjusted along with default values:
+
+| Parameter | Explanation | Type | Default Value |
+| --------- | ----------- | ---- | ------------- |
+| `number_of_replicas` | Number of index replicas | `int` | `0` |
+| `shard_count` | Number of primary shards | `int` | `2` |
+| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `16` |
+| `shard_sizing_iterations` | Number of indexing querying iterations to run | `int` | `25` |
+| `shard_sizing_queries` | Number of queries of each type to run for each iteration | `int` | `20` |
+
 ### 4) elasticlogs-1bn-load
 
 This challenge indexes 1 billion events into a number of indices of 2 primary shards each, and results in around 200GB of indices being generated on disk. This can vary depending on the environment. It can be used give an idea of how max indexing performance behaves over an extended period of time.
 
+The table below shows the track parameters that can be adjusted along with default values:
+
+| Parameter | Explanation | Type | Default Value |
+| --------- | ----------- | ---- | ------------- |
+| `number_of_replicas` | Number of index replicas | `int` | `0` |
+| `shard_count` | Number of primary shards | `int` | `2` |
+| `bulk_indexing_clients` | Number of bulk indexing clients/connections | `int` | `20` |
+| `disk_type` | Type of disk used. If disk_type is not `ssd`, a single merge scheduler thread will be used | `string` | `ssd` |
```

Comment:
Maybe change the wording so it is clear that the track will setup Elasticsearch accordingly? Earlier versions of ES did this automatically (not anymore) and users might be confused.